ci-operator/config/openshift/cluster-logging-operator/openshift-cluster-logging-operator-master.yaml

```diff
@@ -55,7 +55,7 @@ build_root:
   image_stream_tag:
     name: release
     namespace: openshift
-    tag: golang-1.14
+    tag: golang-1.16
```

Comment:
This change may be ok but you need to change line 49 and check the tag for that builder exists here https://console-openshift-console.apps.ci.l2s4.p1.openshiftapps.com/

Reply:
@jcantrill  done: [openshift-cluster-logging-operator-master.yaml#L49](https://github.com/openshift/release/blob/master/ci-operator/config/openshift/cluster-logging-operator/openshift-cluster-logging-operator-master.yaml#L49)